src/screens/Dashboard/DushboardCalendarView/DushboardCalendarView.tsx

```diff
@@ -1,3 +1,43 @@
+import { Box, Typography } from '@mui/material';
+import moment, { Moment } from 'moment';
+import { MomentDateTimeFormat } from '../../../models/moment.models';
+
+const AMOUNT_OF_DAY = 42;
```

Comment:
I'd suggest renaming this const in a way to explain better what kind of number it is.